src/data/optimism/velodromeStableLpPools.json

```diff
@@ -1,4 +1,24 @@
 [
+  {
+    "name": "velodrome-usd+-dai+",
+    "address": "0x69C28d5BbE392eF48C0dC347C575023dAf0CD243",
+    "gauge": "0xFc73bD75f1493f39de41ff41C20AF69C7563F33a",
+    "decimals": "1e18",
+    "chainId": 10,
+    "beefyFee": 0.095,
+    "lp0": {
+      "address": "0x73cb180bf0521828d8849bc8CF2B920918e23032",
+      "oracle": "tokens",
+      "oracleId": "oUSD+",
+      "decimals": "1e6"
+    },
+    "lp1": {
+      "address": "0x970D50d09F3a656b43E11B0D45241a84e3a6e011",
+      "oracle": "tokens",
+      "oracleId": "oUSD+",
```

Comment:
why not `DAI+`?

Reply:
DAI+ price is fetched in fetchWrappedAaveTokenPrices.js here:
```
  optimism: [
    [
      {
        symbol: 'oUSD+',
      },
      wUSDplus,
    ],
    [DAIplus, wDAIplus],
  ],
```
Using a new oracleId breaks the APY calculation as the AMM price cannot be determined.
I just followed the pattern from other DAI+ pools here like the Ramses USD+/DAI+ pool:
``` {
    "name": "ramses-usd+-dai+",
    "address": "0xeb9153afBAa3A6cFbd4fcE39988Cea786d3F62bb",
    "gauge": "0x88d8D2bDC4f12862FbaBEA43cEc08B8FCD2234Da",
    "decimals": "1e18",
    "chainId": 42161,
    "lp0": {
      "address": "0xe80772Eaf6e2E18B651F160Bc9158b2A5caFCA65",
      "oracle": "tokens",
      "oracleId": "oUSD+",
      "decimals": "1e6"
    },
    "lp1": {
      "address": "0xeb8E93A0c7504Bffd8A8fFa56CD754c63aAeBFe8",
      "oracle": "tokens",
      "oracleId": "oUSD+",
      "decimals": "1e18"
    }
  }
```